client/extensions/woocommerce/state/ui/products/api-plan/actions.js

```diff
@@ -9,26 +13,30 @@ import {
 	WOOCOMMERCE_EDIT_PRODUCT_APIPLAN_STEP_END,
 } from '../../../action-types';
 
-/* TODO: Create the plan based on our current edits.
-export function createApiPlanForEdits( productCategoryEdit, productEdits, variationEdits ) {
-	const plan = [];
+export function createApiPlanForEdits( productEdits, variationEdits = null, productCategoryEdits = null ) {
+	// TODO: categories.creates
+	// TODO: categories.updates
+	// TODO: categories.deletes
 
-	// TODO: sequentially go through edit state and create steps.
-	...categories.creates
-	...categories.updates
-	...categories.deletes
+	const productCreateEdits = productEdits.creates || [];
+	const productCreateSteps = productCreateEdits.map( product => {
+		return {
+			name: translate( 'Creating %(product)s', { args: { product: product.name } } ),
```

Comment:
Do you plan to display this in the UI somewhere? Or is it just for debugging purposes?